.rspec

```diff
@@ -1,2 +1,3 @@
 --require spec_helper
---color
\ No newline at end of file
+--color
+--tag ~sudo
```

Comment:
Excluding sudo tag in `.rspec` should not be necessary with the addition `filter_run_excluding` in  `spec_helper`.

Reply:
Woops good call - fixed!